### ISSUE TITLE
question

### ISSUE THREAD

**Issue by hzzzt** [fgjertsen/NLEQ#1]
If I solve a nonlinear equation with 107 unknowns and 107 equations, what should I change

**Comment by fgjertsen** [comment]
Hi. Thank you for the question.

With the dimensionality being 107, it's a much larger than the examples I've used to test the methods, but I've intended for them to be generalized and work for any size of problem, significantly larger than 107 for that matter.

You don't have to change anything in the methods (hopefully), but you will have to provide
1) a handle to a function (returning a 107x1 vector), with your set of equations, on the form f(x)=0
2) a handle to a function (returning a 107x107 matrix), with your Jacobian, J(x). This is optional for Broydens method but mandatory for Newtons method, and I expect it's costly/tedious/"impossible" to derive all elements of this matrix (?).
